lib/honeybadger/plugins/sidekiq.rb

```diff
@@ -24,9 +24,10 @@ def call(worker, msg, queue)
           if defined?(::Sidekiq::VERSION) && ::Sidekiq::VERSION > '3'
             ::Sidekiq.configure_server do |sidekiq|
               sidekiq.error_handlers << lambda {|ex, params|
-                return if params['retry'.freeze] && params['retry_count'.freeze].to_i < config[:'sidekiq.attempt_threshold'].to_i
+                job = params[:job]
+                return if job['retry'.freeze] && job['retry_count'.freeze].to_i < config[:'sidekiq.attempt_threshold'].to_i
                 opts = {parameters: params}
-                opts[:component] = params['wrapped'.freeze] || params['class'.freeze] if config[:'sidekiq.use_component']
+                opts[:component] = params['wrapped'.freeze] || job['class'.freeze] if config[:'sidekiq.use_component']
```

Comment:
No idea about `wrapped`, maybe that needs to be changed as well.

Reply:
I think that we should assume that `'wrapped'` (if it exists) exists in the same location as `'class'`, `'retry'`, etc.